### DISCUSSION TITLE
B7WEB - AVALIAÇÕES/EXPÊRIENCIAS

### DISCUSSION THREAD

**Discussion by carlojos** [frontendbr/forum#2285] [Discussão]
Fala galera, gostaria de de avaliações/experiências com o curso B7WEB do Bonieky Lacerda!
Tô pensando em iniciar esse curso dele...

**Comment by includebrasil** [comment]
Cara, hoje tem muito curso na WEB que chega a embolar a cuca do cabra, é preciso saber procurar! Do meu ponto de vista o que diferencia cada curso é a qualidade do ensino (Didática) e o suporte.

Não tenho dúvidas que o Bonieky Lacerda tenha uma boa didática, mas eu tenho observado que o suporte é horrível! O que me faz pensar que é uma empresa de uma pessoa só (modo de falar). :-)

Eu vinha sondando eles, e percebi essa demora nas respostas, inclusive nas mídias sociais onde nunca fui respondido.  Questionei via e-mail e recebi uma resposta marota, informando que a demanda era alta.

Comprei na sexta feira B7Web Premium, eles me informaram que eu receberia o acesso na segunda (HOJE 02/05/2022), dia que o curso começou e até agora NADA! Recebi um e-mail com um vídeo informando como vai funcionar o curso (nesse vídeo ele ensina algumas coisas que é preciso ter acesso ao painel), mandei um e-mail no período da manhã e até agora NADA, mandei outro agora de tarde e NADA também.

Me organizei para fazer o curso no período das manhãs. Resumindo, perdi um dia do curso!

Chato, vi q ñ começou muito bem e tô aqui com o dedo no gatilho para pedir o reembolso.

**Comment by felquis** [comment]
Não conheço a B7WEB, mas se você acredita que pode te ajudar, parece ser confiável, e está dentro do seu orçamento. Qual é o pior que poderia acontecer? Vai fundo.

**Comment by theandersonn** [comment]
Fiz um curso do Bonieky na udemy {já faz um tempinho} e gostei muito da didática. 
Ainda não conheço a plataforma dele. Recomendo pela didática.

Nessa linha de estudos recomendo tbm a origamid https://www.origamid.com/
Já fiz alguns cursos lá.

**Comment by 52136werT** [comment]
[69718548](https://maximax.cz/) PLNÍME VAŠE OČEKÁVÁNÍ
Plníme vaše očekávání. V každém případě jsme vždy na vaší straně. Vámi objednané produkty se snažíme poslat co nejdříve, protože my sami neradi čekáme.Při výběru produktů se řídíme zásadou tří hlavních pravidel kvalita, design a pohodlí. Také svou práci velmi milujeme. Dali jsme dohromady tým skvělých lidí se zajímavými zkušenostmi, který se rozhodl změnit způsob, jakým lidé zařizují své domovy.

**Comment by 52136werT** [comment]
https://maximax.cz/content/4-o-nas-
